source/fab/application.py

```diff
@@ -87,6 +93,8 @@ def run(self, source: Path):
                 print('    found in: ' + str(filename))
                 print('    depends on: ' + str(fortran_db.depends_on(unit)))
 
+        self._queue.check_queue_done()
```

Comment:
Even though we don't expect to keep the debug output waiting for the queue to empty should probably happen before it.

Reply:
True, done.